src/main/java/duke/command/ByeCommand.java

```diff
@@ -0,0 +1,29 @@
+package duke.command;
+
+import duke.Duke;
+import duke.task.TaskList;
+import duke.GUI;
+
```

Comment:
imports here don't seem lexicographically ordered. i've omitted other import ordering related comments from this review